src/set.cr

```diff
@@ -71,6 +71,19 @@ struct Set(T)
     self
   end
 
+  # If the *object* is not in the set, adds it and returns `true`,
```

Comment:
The description should not start with a conditional clause, makes it hard to understand.
```suggestion
  # Adds *object* to the set and returns `true` on success and `false` if the value was already in the set.
```